odoo/addons/base/models/res_partner.py

```diff
@@ -190,14 +190,14 @@ def default_get(self, default_fields):
                 values['type'] = None
         return values
 
-    name = fields.Char(index=True, default_export_compatible=True)
-    complete_name = fields.Char(compute='_compute_complete_name', store=True, index=True)
-    date = fields.Date(index=True)
+    name = fields.Char(index='trigram', default_export_compatible=True)
+    complete_name = fields.Char(compute='_compute_complete_name', store=True, index='trigram')
+    date = fields.Date()
     title = fields.Many2one('res.partner.title')
     parent_id = fields.Many2one('res.partner', string='Related Company', index=True)
     parent_name = fields.Char(related='parent_id.name', readonly=True, string='Parent name')
     child_ids = fields.One2many('res.partner', 'parent_id', string='Contact', domain=[('active', '=', True)])
-    ref = fields.Char(string='Reference', index=True)
+    ref = fields.Char(string='Reference')
```

Comment:
Are you sure for this one? 
It is still use for https://github.com/odoo/odoo/blob/3ec29a8e422e1696ac1e9e05c66d5a76137c9f4e/addons/mail_plugin/controllers/mail_plugin.py#L197 by example.